widget/storybook/src/components/Header.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import type { Meta, StoryObj } from '@storybook/react';
```

Comment:
Make sure you are covering all the use cases (props) in your story.

Reply:
Sure let me update story after you confirmed props.

Reply:
Considering that there is no added props in final changes, there is no extra use case to be covered.